emscripten/Makefile

```diff
@@ -0,0 +1,21 @@
+CC      = emcc
+
+CFLAGS  = -O3 -W -Wall -Wextra -DNOCRYPT -DUSE_FILE32API -Wno-unused-parameter
+CFLAGS += --llvm-lto 3 --closure 1 -s USE_ZLIB=1 -s INVOKE_RUN=0 -s NO_EXIT_RUNTIME=1 --pre-js luna-utils.js
+
+VPATH   = ../ ../minizip-1.1
+
+OUTPUT := luna
+
+all: $(OUTPUT).js
+
+%.bc: %.c
+	$(CC) $(CFLAGS) $< -o $@
+
+$(OUTPUT).js: luna.bc zip.bc ioapi.bc DES.bc
+	$(CC) $(CFLAGS) $(LFLAGS) -o $@ $^
+
+clean:
+	$(RM) -r $(OUTPUT).js* $(OUTPUT).data *.bc
```

Comment:
Using *.bc here but listing all files seperately above is inconsistent and might cause bugs